Velaptor/NativeInterop/Services/OpenGLService.cs

```diff
@@ -9,25 +9,51 @@ namespace Velaptor.NativeInterop.Services;
 using System.Drawing;
 using System.Numerics;
 using OpenGL;
+using Silk.NET.OpenGL;
 using Velaptor.OpenGL;
+using Velaptor.Services;
 
 /// <summary>
 /// Provides OpenGL helper methods to improve OpenGL related operations.
 /// </summary>
 internal sealed class OpenGLService : IOpenGLService
 {
+    // ReSharper disable InconsistentNaming
+#pragma warning disable SA1310
+    private const int API_ID_RECOMPILE_FRAGMENT_SHADER = 2;
+    private const int API_ID_RECOMPILE_VERTEX_SHADER = 131218;
+#pragma warning restore SA1310
+
+    // ReSharper restore InconsistentNaming
     private readonly IGLInvoker glInvoker;
+    private readonly IDotnetService dotnetService;
+    private readonly ILoggingService loggingService;
+    private DebugProc? debugCallback;
+
+    // ReSharper disable once MemberInitializerValueIgnored
+    private GL gl = null!;
```

Comment:
Please remove this field.  This is not needed anymore because it has been replaced by `IGLInvoker`.
```suggestion
```